Loop/Managers/DeviceDataManager.swift

```diff
@@ -224,10 +264,41 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
         if status.batteryRemainingPercent == 0 {
             NotificationManager.sendPumpBatteryLowNotification()
         }
+    }
+
+    private func uploadDeviceStatus(pumpStatus: NightscoutUploadKit.PumpStatus? /*, loopStatus: LoopStatus */) {
+
+        guard let uploader = remoteDataManager.nightscoutUploader else {
+            return
+        }
+
+
+        // Gather UploaderStatus
+        let uploaderDevice = UIDevice.currentDevice()
 
-        remoteDataManager.nightscoutUploader?.handlePumpStatus(status, device: device.deviceURI)
+        let battery: Int?
+        if uploaderDevice.batteryMonitoringEnabled {
```

Comment:
Do we need to enable device battery monitoring when NightscoutUploader becomes authenticated?